docs/development-roadmap.stories.mdx

```diff
@@ -0,0 +1,36 @@
+<Meta
+  title="Documentation/Development Roadmap"
+/>
+
+# Sage Carbon public roadmap
+
+> **Disclaimer**: We operate in a dynamic commercial environment, and things are subject to change. The information provided is intended to outline the general direction of the library. It's intended for informational purposes only. We may decide to add/remove new items at any time depending on our capability to deliver while meeting our quality standards. The development, releases, and timing of any features or functionality of Carbon remain at the sole discretion of Sage. The roadmap does not represent a commitment, obligation, or promise to deliver at any time.
+
+## Upcoming activity
+Listed below is the activity we have on our backlog. 
+### Move experimental folder
+Move the experimental components into the main components folder and remove the experimental folder
+### Internationalisation 
+Implement the proposal in the [i18n RFC](https://github.com/Sage/carbon/blob/master/rfcs/text/i18n.md)
+### TypeScript Support 
+ At a minimum we should support TypeScript by providing typings files for every component and enabling support in our build pipeline. 
```

Comment:
I don't think theres anything we need to do in the build pipeline, we already support `d.ts` files.
```suggestion
 At a minimum we should support type definitions for every component.
```